src/TodoListItem.js

```diff
@@ -1,9 +1,12 @@
-import React from "react"
+import React from "react";
 
-function TodoListItem({title}) {
-    return(
-        <li>{title}</li>
-    )
+function TodoListItem({ id, title, onRemoveTodo}) {
+  return (
+    <li>
+      {title}
```

Comment:
Let's add some spacing between the title and the remove button. Right now they're next to each other:
- List Item`Remove`